config/dynamicconfig/development-xdc.yaml

```diff
@@ -0,0 +1,57 @@
+#frontend.enableClientVersionCheck:
+#- value: true
+#  constraints: {}
+#history.persistenceMaxQPS:
+#- value: 3000
+#  constraints: {}
+#frontend.persistenceMaxQPS:
+#- value: 3000
+#  constraints: {}
+#frontend.throttledLogRPS:
+#- value: 20
+#  constraints: {}
+#history.defaultActivityRetryPolicy:
+#- value:
+#    InitialIntervalInSeconds: 1
+#    MaximumIntervalCoefficient: 100.0
+#    BackoffCoefficient: 2.0
+#    MaximumAttempts: 0
+#history.defaultWorkflowRetryPolicy:
+#- value:
+#    InitialIntervalInSeconds: 1
+#    MaximumIntervalCoefficient: 100.0
+#    BackoffCoefficient: 2.0
+#    MaximumAttempts: 0
+#system.secondaryVisibilityWritingMode:
+#  - value: "off"
+#    constraints: {}
+#system.enableReadFromSecondaryVisibility:
+#  - value: false
+#    constraints: {}
+#system.enableParentClosePolicyWorker:
+#  - value: true
+system.enableEagerWorkflowStart:
+  - value: true
+frontend.enableUpdateWorkflowExecution:
+  - value: true
+frontend.enableUpdateWorkflowExecutionAsyncAccepted:
+  - value: true
+frontend.workerVersioningDataAPIs:
+  - value: true
+frontend.workerVersioningWorkflowAPIs:
+  - value: true
+frontend.accessHistoryFraction:
+  - value: 1.0
+frontend.adminDeleteAccessHistoryFraction:
+  - value: 1.0
+system.enableNexus:
+  # enableNexus also controls transition history which can't be enabled now
+  - value: false 
```

Comment:
cc @yux0 I add a separate dev config file for xdc so that I can keep nexus and new transition history stuff disabled for now.